src/main/common/helpers/paymentPlanHelper.ts

```diff
@@ -0,0 +1,121 @@
+
+import { Moment } from 'moment'
+
+import { Frequency } from 'common/frequency/frequency'
+import { ResponseType } from 'claims/models/response/responseType'
+import { PaymentPlan } from 'common/payment-plan/paymentPlan'
+import { Claim } from 'claims/models/claim'
+import { FullAdmissionResponse } from 'claims/models/response/fullAdmissionResponse'
+import { PartialAdmissionResponse } from 'claims/models/response/partialAdmissionResponse'
+
+import { DraftClaimantResponse } from 'features/claimant-response/draft/draftClaimantResponse'
+import { ResponseDraft } from 'features/response/draft/responseDraft'
+import { PaymentIntention } from 'shared/components/payment-intention/model/paymentIntention'
+
+import { RepaymentPlan as ClaimPaymentPlan } from 'claims/models/response/core/repaymentPlan'
+import { PaymentPlan as DraftPaymentPlan } from 'main/common/components/payment-intention/model/paymentPlan'
+import { PaymentPlan as FormPaymentPlan } from 'shared/components/payment-intention/model/paymentPlan'
+
+export class PaymentPlanHelper {
```

Comment:
damn could have done with this in the ccj claimant response confirmation page code

Reply:
do we need a techDebt story to refactor the CCJ now?